akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventWriter.scala

```diff
@@ -257,6 +314,98 @@ private[akka] object EventWriter {
               Behaviors.same
           }
 
+        def handleWrite(repr: PersistentRepr, replyTo: ActorRef[StatusReply[WriteAck]]): Unit = {
```

Comment:
I'm quite sure it's not handling all concurrent combinations yet, and maybe there are cases where it's not preserving the right order?